### PR TITLE
Added support for Cisco AireOS Wireless LAN Controllers

### PR DESCRIPTION
**Pull request by AirManic**
Offering up my work to add support for Cisco AireOS WIreless LAN Controllers (WLC).

### CODE REVIEW

**Comment by jamiecaesar** [comment]
This looks awesome and I appreciate you for putting this together and contributing it to this repository.  I plan to dig into this more later tonight.   I realized I had a few things in the devel branch that  were never merged into master, so I did that and it caused a merge conflict on one file (sessions.py).   I think I can deploy this PR into a new branch and sort that out, or if it looks easy on your side to change the destination to the devel branch, that would be helpful as well.

In short, I just want to make sure I can deploy this somewhere that I can make sure all of the Sphinx auto-generating documentation stuff is in place before merging into master.

Thanks again!

**Comment by AirManic** [comment]
Not a programmer at heart, so I am learning.  Had to sort of hack in how to detect it is a Cisco AireOS WLC.
BTW, actually my first experience with github, so hope I did it correctly.
Once you accept, I will pull your master back into my master and compare to my branch(es).  Have more coming on this front, but what I submitted was a solid chunk to get started.
.:|:.:|:. Gordon Rogier, CCIE-Wireless #26419 | Technical Leader, Customer eXperience | 972.358.1580<mailto:972.358.1580> | grogier@cisco.com<mailto:grogier@cisco.com>

**Comment by jamiecaesar** [comment]
Also, please let me know if you have any problems accessing the new branch.

**Comment by jamiecaesar** [comment]
Made some fixes to the AireOS branch, based on testing against my home WLC, and merged it into the master branch.  @AirManic -- please let me know if you find any issues with your scripts after pulling master.

**Comment by AirManic** [comment]
Will check out a bit later and let you know.  Sounds great.  .:|:.:|:. Gordon Rogier
